platform/feature-api/api/pod-api/src/api.ts

```diff
@@ -199,6 +182,31 @@ export interface PolyLifecycle {
     startFeature(id: string, background: boolean): Promise<void>;
 }
 
+/**
+ * `Endpoint` is the API features communicate with in order to perform fetch requests
+ */
+export interface Endpoint {
+    /**
+     * Perform a http post request via the endpoint in the pod
+     * @param body the necessary content of the call
+     * @returns an empty response or throws an error if something goes wrong
+     * @throws if an unsupported request goes through, if an endpoint is not reached or if a user denies a request
+     */
+    send(
+        endpointId: string,
```

Comment:
Additionally, we're using endpointID as an argument, but it might better to make it just an instance variable, to be included in the constructor of whatever instances implement this interface. After all, what we're going to have are instances that should *know* their own ID, right?
Again, something that could be done in a refinement, later.